src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = new ManufacturerDaoImpl();
+        Manufacturer manufacturer1 = new Manufacturer(null, "Lincoln", "USA");
+        Manufacturer manufacturer2 = new Manufacturer(null, "Ford", "USA");
+        Manufacturer manufacturer3 = new Manufacturer(null, "Audi", "Germany");
+        Manufacturer manufacturer4 = new Manufacturer(null, "BMW", "Germany");
+        manufacturerDao.create(manufacturer1);
+        manufacturerDao.create(manufacturer2);
+        manufacturerDao.create(manufacturer3);
+        manufacturerDao.create(manufacturer4);
+
+        manufacturerDao.delete(16L);
+        manufacturerDao.delete(22L);
+        manufacturerDao.delete(24L);
+        manufacturerDao.delete(27L);
+
+        System.out.println(manufacturerDao.get(23L));
```

Comment:
Then repeat get methods here. And compare results before deletions and after
```suggestion
        
        Optional<Manufacturer> deletedManufacturer1 = manufacturerDao.get(manufacturer1.getId());
        List<Manufacturer> manufacturersAfterDeletion = manufacturerDao.getAll();
        
        System.out.println(manufacturerDao.get(//useIdFromManufacturer));
```